app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class homecontroller{ $th }
+
+class homecontrollerwa { $th }
```

Comment:
__[code_sniffer]__
* Each class must be in a file by itself
* Class name "homecontrollerwa" is not in camel caps format
* Opening brace of a class must be on the line after the definition
* The closing brace for the class must go on the next line after the body
* Closing brace must be on a line by itself

<!-- Comment by [SideCI](https://sideci.com) -->